pages/hpc/login.md

```diff
@@ -43,7 +43,7 @@ Using the SSH protocol, you can connect and authenticate to CCR login nodes with
 
 ## Generate new SSH key
 
-The first time you want to login to CCR's login servers, you'll need to generate a new SSH key pair on your local machine. After you generate
+The first time you want to login to CCR's login servers on a new device, you'll need to generate a new SSH key pair on your local machine. After you generate
```

Comment:
I don't love this wording overall.  Considering the previous paragraph, I recommend this instead:
```suggestion
Prior to using SSH to connect to CCR's login servers, you'll need to generate a new SSH key pair on your local machine. If you connect to CCR from multiple devices, we recommend generating a separate SSH key pair for each device.  After you generate
```